app/src/newdocs/content/tutorials.mdx

```diff
@@ -14,96 +14,30 @@ import Postman from './images/tutorials/postman.png'
-
-Example using python + requests:
-
-<CodeSnippet language='python'>{PythonRequests}</CodeSnippet>
-
-Example using `curl`:
-
-<CodeSnippet language='bash'>
-    {`curl -i -X POST -H "Authorization: Bearer MY-SESSION-TOKEN" -d "{\"foo\":\"hello\",\"bar\":24.5}" https://www.streamr.com/api/v1/streams/MY-STREAM-ID/data`}
-</CodeSnippet>
-
-### Create a stream
-
-If you’re already signed in, click Core in the top navigation. If you’re not, sign in and you will be taken to your User Page. From here, go to the Streams tab, and you will see a view something like this one.
-
-<div className={docsStyles.centered}>
-  <img src={CreateStream} />
-</div>
-
-### Get your Stream ID
-From here, click Create Stream. You will be taken to the stream Setup view. First you need to name your stream and write a basic description. The Stream ID will be created for you. Click to copy the ID to clipboard. You’ll need this in order to produce events to your stream via the API. Click Save and Exit to return to your User Page.
-
-<div className={docsStyles.centered}>
-  <img src={StreamSetup} />
-</div>
-
-### Get your API key
-
-Here you’ll notice the new, but empty stream you’ve created. From your profile menu (top right), choose API Keys. You need this to authenticate to the API.
-
-<div className={docsStyles.centered}>
-  <img src={GetYourAPIKey} />
-</div>
-
-From **Settings > API Keys**, click to add a new key, and give it a name like Tutorial.  Click the meatball menu to copy the new key and click Save Changes to return to your User Page.
-
-<Warning>
-
-**Only use your tutorial API keys for testing and development.** This ensures you don't accidentally modify a live stream or canvas.
-
-</Warning>
+### Publish to a stream
 
-<div className={docsStyles.centered}>
-  <img src={SettingsAPICredentials} />
-</div>
+The easiest way to push data to a stream is to use the one of our <Link to={links.newdocs.SDKs}>SDKs</Link>, currently available in [Javascript](https://github.com/streamr-dev/streamr-client-javascript), [Java](https://github.com/streamr-dev/streamr-client-java) and a community contributed version in [Python](https://github.com/streamr-dev/streamr-client-python). Client libraries for other languages are on the roadmap.
 
-### Publish to a stream
+Here is an example of publishing data to a stream using Streamr Javascript client:
 
-The easiest way to push data to a stream is to use the [JavaScript client](https://github.com/streamr-dev/streamr-client-javascript), which works in node.js as well as the browser. Client libraries for other languages are on the roadmap. 
+<CodeSnippet language='javascript' wrapLines showLineNumbers >{ClientPub}</CodeSnippet> 
 
 You can also try it locally in your browser. Just open this [JS Bin](https://jsbin.com/muqarukubu/edit?js,output), and replace <kbd>MY-STREAM-ID</kbd> and <kbd>MY-API-KEY</kbd> with your Stream ID and API key. Your browser will start producing data to your Stream!
```

Comment:
This jsbin is using version 0.10.1 (very old client)
`<script src="https://unpkg.com/streamr-client@0.10.1/dist/streamr-client.min.js"></script>` could you create a new JS bin using the latest version?

https://unpkg.com/browse/streamr-client@2.1.1/dist/

`https://unpkg.com/browse/streamr-client@2.1.1/dist/streamr-client.web.min.js`